docs/script_daily.md

```diff
@@ -0,0 +1,103 @@
+# Shell Script for Checking for new bugs if results are stored in daily runs
+
+The following is a concise Shell script that can be used in a Jenkins or any
+other CI engine. This will exit with return status `1`, failing the job, if
+new bugs are introduced into the codebase.
+
+~~~{.sh}
```

Comment:
If we would like to specify the language of a code block we have to use the following forms: `~~~bash` or ` ```bash`.
For more information see: https://help.github.com/articles/creating-and-highlighting-code-blocks/
This will enable syntax highlight on the code block.

Reply:
I think this is the Doxygen highlighting syntax. Do we still use the Doxygen on the project (not the user guide) locally at all?

Reply:
We are using doxygen to generate the documentation into the released package (not only for the user guide).